csrc/utils.cpp

```diff
@@ -9,6 +9,7 @@
 #include <utils.h>
 
 #include <c10/util/string_view.h>
+#include <nvrtc.h>
```

Comment:
Why is this necessary?

Reply:
Because we are calling nvrtc APIs directly now, instead of going through the pytorch stub `at::globalContext().getNVRTC()`, so we'll need to have nvrtc header directly.

Reply:
Yes, but does utils.cpp need it? I looked at the file but couldn't see where it might be necessary.

Reply:
:exploding_head: ha, good catch... Let me double check the two headers added by me.